### PR TITLE
ANN: Fix saving documents in cargo check annotator

### PR DESCRIPTION
**Pull request by mchernyavsky**

### CODE REVIEW

**Comment by vlad20012** [comment]
With this fix Trailing Spaces Stripper totally stops working if cargo check is enabled

**Comment by mchernyavsky** [comment]
Fixed.

File: src/test/kotlin/org/rustSlowTests/RsCargoCheckAnnotatorTest.kt [hunk 1]
```diff
@@ -131,6 +133,25 @@ class RsCargoCheckAnnotatorTest : RsWithToolchainTestBase() {
         }
     }
 
+    fun `test strip document later reflection`() {
```
Comment (Undin) [inline src/test/kotlin/org/rustSlowTests/RsCargoCheckAnnotatorTest.kt]: If I understand correctly, this test checks that reflection doesn't throw exceptions
If yes, let's add comment about it

**Pull request merged**